code/modules/martial_arts/combos/judo/armbar.dm

```diff
@@ -0,0 +1,17 @@
+/datum/martial_combo/judo/armbar
+	name = "Armbar"
+	steps = list(MARTIAL_COMBO_STEP_GRAB, MARTIAL_COMBO_STEP_DISARM,MARTIAL_COMBO_STEP_GRAB, MARTIAL_COMBO_STEP_HARM)
+	explaination_text = "place an opponent on the ground into an armbar, immobilizing them"
+	combo_text_override = "Grab, swap hands, disarm, grab, swap hands, harm"
+
+/datum/martial_combo/judo/armbar/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
+	if(target.IsWeakened() || !IS_HORIZONTAL(target))
+		return MARTIAL_COMBO_FAIL
+	target.visible_message("<span class='warning'>[user] puts [target] into an armbar!</span>", \
+						"<span class='userdanger'>[user] wrestles you into an armbar!</span>")
+	playsound(get_turf(user), 'sound/weapons/slashmiss.ogg', 40, 1, -1)
+	target.apply_damage(45, STAMINA)
+	target.SetImmobilized(5 SECONDS)
+	target.SetKnockDown(5 SECONDS)
```

Comment:
Not sure if setting them to knockeddown for 5 seconds does anything, since they are also immobilized for 5 seconds

Reply:
Knockdown and immobilized are completely separate things

Reply:
This was done to A) root the target AND B) keep them on the floor, as required for the wheel throw combo.